pkg/config/config_test.go

```diff
@@ -1336,3 +1336,9 @@ func TestAutoScalerConfig(t *testing.T) {
 		conf.UseAutoScaler = false
 	})
 }
+
+func TestGetGlobalConfigDetail(t *testing.T) {
+	d, err := GetGlobalConfigDetail()
+	require.NoError(t, err)
+	require.NotNil(t, d)
```

Comment:
Maybe we should check the content of `d`?

Reply:
I am not sure how to check it, so I only check `default value == current value`. Should I check more?

Reply:
Maybe something like this:
```patch
diff --git a/pkg/config/config_test.go b/pkg/config/config_test.go
index e04a06e72d..04e60c88ad 100644
--- a/pkg/config/config_test.go
+++ b/pkg/config/config_test.go
@@ -1344,5 +1344,10 @@ func TestGetGlobalConfigDetail(t *testing.T) {
        require.True(t, len(d) > 0)
        for _, v := range d {
                require.Equal(t, v.Value, v.DefaultValue)
+               require.NotEqual(t, v.Name, "")
+               switch v.Name {
+               case "path":
+                       require.Equal(t, v.Value, "/tmp/tidb")
+               }
        }
 }
```